website/src/content/docs/pt-br/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Biome
+head:
+  - tag: title
+    content: Biome
+template: splash
+description: Formate, verifique erros e muito mais em uma fração de segundo.
+hero:
+  title: Uma toolchain para seu projeto web
+  tagline: Formate, verifique erros e muito mais em uma fração de segundo.
+  image:
+    alt: Biome, Toolchain of the web
```

Comment:
Let's also translate the `alt`

Reply:
Currently, I can't think of an appropriate translation for this. Do you have any suggestions?

Reply:
My suggest is something like "Biome, Conjunto de ferramentas da web".

What do you think?

Reply:
LGTM!